### PR TITLE
Upgrade all old asides in tutorials & simplify some

### PR DESCRIPTION
**Pull request by e111077**
I also took a stab at trying to simplify some of the longer asides

### CODE REVIEW

File: packages/lit-dev-content/site/tutorials/content/wc-to-lit/11.md [hunk 1]
```diff
@@ -58,9 +58,9 @@ What's changed?
 
 The `css` tag function allows you to define CSS tagged template literals, which enable math, templating, and other features under the hood.
 
-{% aside "info" %}
+{% aside "info" "no-header" %}
```
Comment (arthurevans) [inline packages/lit-dev-content/site/tutorials/content/wc-to-lit/11.md:61]: Why does this one use a macro while the other ones use the tag directly?

**Comment by arthurevans** [comment]
Ack, sorry, totally missed this.

File: packages/lit-dev-content/site/tutorials/content/wc-to-lit/11.md (hunk 1 above)
Comment (e111077) [reply packages/lit-dev-content/site/tutorials/content/wc-to-lit/11.md:61]: I meant to update all of them ahaha thanks for the catch

**Comment by e111077** [comment]
ping on this one if you've got the time

File: packages/lit-dev-content/site/tutorials/content/async-directive/03.md [hunk 2]
```diff
@@ -40,30 +40,18 @@ Last, update the render method to use `format`:
...
+{% aside  "info" %}
 
-Note that standard `Directive`s like what we created with here have a lot more
-powers than we're using so far: they are stateful between renders and can access
-the underlying DOM they are associated with. See [Custom
-directives](/docs/templates/custom-directives/#creating-class-based-directives)
-for more details.
-
-<br>
+Directives can keep state, operate outside of Lit's lifecycle, and safely access
```
Comment (arthurevans) [inline packages/lit-dev-content/site/tutorials/content/async-directive/03.md:45]: I feel like this one needs updates, but I'm not sure these edits retain enough of the original. Can we just turn this into a no-header aside with the original text for now?

File: packages/lit-dev-content/site/tutorials/content/async-directive/05.md [hunk 3]
```diff
@@ -40,15 +40,16 @@ handle, and use that to start the timer only once:
...
 
-<aside class="positive">
+{% aside  "positive" %}
 
-The default implementation of `update` simply calls `render`, but since `update`
-is [_not_ called during
-SSR](https://lit.dev/docs/templates/custom-directives/#differences-between-update()-and-render()),
-`update` is the right spot to start/subscribe to asynchronous tasks we don't
-want running on the server.
+Start or subscribe to asynchronous tasks that you don't want running in the
```
Comment (arthurevans) [inline packages/lit-dev-content/site/tutorials/content/async-directive/05.md:45]: Same as the last one, I think we should leave the original text as a no-header aside for now.

**Review by arthurevans — CHANGES_REQUESTED** [review]
I felt a couple of Kevin's asides should be left intact for now, otherwise LGTM.

**Pull request merged**